osquery/tables/system/darwin/homebrew_packages.cpp

```diff
@@ -55,7 +58,9 @@ std::vector<std::string> getHomebrewVersionsFromInfoPlistPath(
   return results;
 }
 
-Status getHomebrewCellar(fs::path& cellarPath) {
+void packagesFromPrefix(QueryData& results,
+                        const std::string& prefix,
+                        bool userRequested) {
```

Comment:
minor naming nit on `userRequested`, perhaps intent is more clearer with something like `prefixConstraintPresent`? Thoughts?

Reply:
The intent is to log if a user requested path is absent. But I wasn't sure if I'd use it for anything else. Could be `logWarnings` or something

Reply:
There might be a better name than `prefix` too 🤷

Reply:
gotcha -- I like `prefix`, because it's consistent with what homebrew uses